### DISCUSSION TITLE
Cash zipline integrate JS with Bugsnag crash data

### DISCUSSION THREAD

**Discussion by kpgalligan** [touchlab/Kermit#423] [Ideas]

**Comment by luca992** [comment]
I'm looking for a way to report exceptions in a multiplatform compose app (android, jvm, ios, macos, js).
I see js was removed as a target from bugsnag as of 1.2.2. Is it planned to be added back by using zipline to use the js bugsnag package in common code? Or something along those lines? (I haven't really looked into zipline before)

Would that let you add more targets like standard jvm as well? It looks like standard jvm was never supported, just android.
